pandas/util/_decorators.py

```diff
@@ -96,6 +100,25 @@ def deprecate_kwarg(old_arg_name, new_arg_name, mapping=None, stacklevel=2):
     FutureWarning: old='yes' is deprecated, use new=True instead
       warnings.warn(msg, FutureWarning)
     yes!
+
+
```

Comment:
Can you have this many blank lines in a docstring without flake8 complaining?

Reply:
Yes that is correct.  I didn't get any warnings here.